features/extension-mgt/org.wso2.carbon.identity.extension.mgt.feature/resources/extensions/connections/twofactor-custom-authentication/template.json

```diff
@@ -0,0 +1,14 @@
+{
+  "category": "DEFAULT",
+  "description": "Register externally implemented authentication service.",
+  "docLink": "",
+  "templateGroup": "custom-authentication",
+  "displayOrder": 1,
+  "id": "twofactor-custom-authentication",
+  "tags": [ "Custom" ],
+  "image": "",
+  "name": "2FA Authentication",
+  "services": [],
+  "disabled": false,
+  "templateId": "twofactor-custom-authentication"
+}
```

Comment:
Can't we have two items for custom authentication type as idp and authenticator, one of them will have non empty value

Reply:
This is updated with 7d2336487ac11a54b528c7c248af61bb678b98e8